gapis/api/vulkan/api/physical_device.api

```diff
@@ -53,6 +53,9 @@
   @unused ref!SubgroupProperties         SubgroupProperties
   @unused ref!PhysicalDeviceIDProperties IDProperties
 
+  // Vulkan 1.2 core
+  @unused ref!PhysicalDeviceBufferDeviceAddressFeatures PhysicalDeviceBufferDeviceAddressFeatures
```

Comment:
Why do we have both? This field doesn't appear to be used.
Also, add it to the end, so we don't break backwards compatibility.

Reply:
This is not doable in the long term.

It's either we group extensions by version or we scrap that and put them randomly without no specific order. Because we cannot guarantee that our implementation priority will match the Vulkan version.

If backwards compatibility is important we should ensure it with another way.

Reply:
And with this release due to vertex attribute fix, we will break the backward compatibility anyway, so it doesn't matter. We should I believe do all the breaking changes we think of before the next release and do them at once.

Reply:
I've removed the KHR one.

I think in terms of promoted extensions, we can use the core version without KHR unless we need KHR explicitly.